microcosm_flask/fields.py

```diff
@@ -46,3 +49,48 @@ def _deserialize_by_name(self, value, attr, data):
             return getattr(self.enum, value)
         except AttributeError:
             self.fail('by_name', name=value)
+
+
+class TimestampField(Field):
+    """
+    Timestamp valued field, as either a unix timestamp (default) or isoformat string.
+
+    """
+    EPOCH = datetime(1970, 1, 1)
+
+    def __init__(self, use_isoformat=False, *args, **kwargs):
+        self.use_isoformat = use_isoformat
+        super(TimestampField, self).__init__(*args, **kwargs)
+
+    def _serialize(self, value, attr, obj):
+        """
+        Serialize value as a timestamp, either as a Unix timestamp (in float second) or a UTC isoformat string.
+
+        """
+        if value is None:
+            return None
+
+        if self.use_isoformat:
+            return datetime.utcfromtimestamp(value).isoformat()
```

Comment:
@Phylodome is getting some `year is out of range for ` errors on jeremiah right now and I think it uses similar code.  Probably worth our figuring out what that issue is prior to merging.